### PR TITLE
groups: add page

### PR DESCRIPTION
**Pull request by supermamie**
Updates #2213.

----

- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/groups.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# groups
+
+> Print the groups a user is in.
```
Comment (sbrl) [inline pages/linux/groups.md]: Perhaps ....`groups that a user`.... would read better?

File: pages/linux/groups.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- Display for current user:
```
Comment (sbrl) [inline pages/linux/groups.md]: Perhaps `Display the groups of the current user` would read better?

File: pages/linux/groups.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+`groups`
+
+- Display for another user:
```
Comment (sbrl) [inline pages/linux/groups.md]: Perhaps `Display the groups for`..... would read better?

File: pages/linux/groups.md (hunk 1 above)
Comment (supermamie) [reply pages/linux/groups.md]: Do you think "Print group memberships for a user" would be better ?

File: pages/linux/groups.md (hunk 2 above)
Comment (supermamie) [reply pages/linux/groups.md]: Do you think "Print group memberships for the current user" would be better ?

File: pages/linux/groups.md (hunk 3 above)
Comment (supermamie) [reply pages/linux/groups.md]: Do you think "Print group memberships for users" would be better ?
(I'll also modify the doc and add an example with a list of users)

File: pages/linux/groups.md (hunk 2 above)
Comment (sbrl) [reply pages/linux/groups.md]: Go for it!

File: pages/linux/groups.md (hunk 3 above)
Comment (sbrl) [reply pages/linux/groups.md]: Erm not sure here. That implies multiple users - we've got a single user here.

Comment (sbrl) [reply pages/linux/groups.md]: Perhaps `Print group memberships for a specific user` might work instead?

**Pull request merged**